tests/test_utils.py

```diff
@@ -248,5 +249,33 @@ def test_multipart_parallel_upload(self):
     def restore_email_claims(old):
         os.environ['OIDC_EMAIL_CLAIM'] = old
 
+
+@testmode.standalone
+class TestDynamodbUtils(unittest.TestCase):
+
+    def test_get_item(self):
+        """test out functionality of removing the attribute types from a dynamodb response"""
```

Comment:
```suggestion
        """
        Test out functionality of removing the attribute types from a dynamodb response.
        Example: {"foo": {"S": "bar"}} should become {"foo": "bar"}
        """
```
Add a quick clarifying example to the docstring

Reply:
added